sdk/docs/pages/typescript-sdk/connect.mdx

```diff
@@ -0,0 +1,53 @@
+# Connecting to Sui Network
+
+The Sui TypeScript SDK provides a SuiClient class that you can use to connect to a network's JSON-RPC server. Use the `SuiClient` for all JSON-RPC read-only operations. 
+
+To establish a connection to a network, import `SuiClient` from `@mysten/sui.js/client` and pass the relevant URL to the `endpoint` parameter. The following example establishes a connection to Testnet and requests SUI from that network's faucet.
+
+```typescript
+import { SuiClient } from '@mysten/sui.js/client';
+// assign Testnet RPC address
+const testnet = 'https://fullnode.testnet.sui.io:443/';
+// set endpoint value to Testnet RPC URL
+const SuiClient = new SuiClient({ endpoint: testnet });
+// get tokens from the Testnet faucet server
+await suiClient.getCoins({
+	owner: '<OWNER_ADDRESS>',
+});
+```
+
+| Connection object | Full node address | `faucet` location |
```

Comment:
connect objects are being removed from the sdk, maybe we can just use `network` for this column